pint/context.py

```diff
@@ -62,7 +62,7 @@ class Context(object):
 
     """
 
-    def __init__(self, name, aliases=(), defaults=None):
+    def __init__(self, name=None, aliases=(), defaults=None):
 
         self.name = name
```

Comment:
I am not sure how things will work if somebody create two anonymous contexts. Therefore, I think that it would be better to create a random name.

Reply:
I added tests for two anonymous contexts

Reply:
@hgrecco ping - is there anything outstanding?